src/mediaelement.js

```diff
@@ -105,6 +106,14 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.seekTo(start);
         this.media.play();
         end && this.setPlayEnd(end);
+        var my = this;
+        function updateProgress(){
+            my.fireEvent('audioprocess', my.getCurrentTime());
+            if (my.updateProgressAnimating) {
+                requestAnimationFrame(updateProgress);
+            }
+        }
+        this.updateProgressAnimating = requestAnimationFrame(updateProgress);
```

Comment:
It shouldn't be part of the drawer. It should be used in the drawer code. I'm OK for having `requestAnimationFrame` for both Web Audio and Media Element.